### ISSUE TITLE
Review call with La Matrice

### ISSUE THREAD

**Issue by EilidhRoss1** [life-itself/community#165]
I spoke with Pierre-Alexandre Klein from La Matrice last month. He was super nice, very interested in and excited about the work we're doing and really keen to see how La Matrice can connect with/ work with Life Itself.

I've written up a [brief](https://docs.google.com/document/d/1mnU2vLpIM9yH2vHNEHYMktvdASd9jwtdSwl5mJtFAN4/edit#) from the call. It's 6 pages, with a **2 page summary** that would suffice if you don't have the time to read the whole thing. I'd love to set up a call with @rufuspollock and Pierre-Alexandre. 

## Acceptance

* [ ] Rufus has read the [brief](https://docs.google.com/document/d/1mnU2vLpIM9yH2vHNEHYMktvdASd9jwtdSwl5mJtFAN4/edit#)
* [ ] We have decided how to follow up

## Tasks

* [ ] Read the [brief](https://docs.google.com/document/d/1mnU2vLpIM9yH2vHNEHYMktvdASd9jwtdSwl5mJtFAN4/edit#)
* [ ] Rufus talk with Eilidh about how we should follow up. If appropraite:
  * [ ] Eilidh set up follow up call

**Comment by rufuspollock** [comment]
@EilidhRoss1 i'd read the (excellent) summary before. Just left comments. Overall they sound great ... and i don't have a very specific sense of the tangible overlap atm. (though i wasn't on the call).

**Comment by rufuspollock** [comment]
@EilidhRoss1 further reflections: to follow up, normally one thing i look to if there is a significant opportunity for alignment or collaboration is to start working on an SCQH with the potential collaborator. This often proves valuable in itself and also gives a much clearer sense of collaboration. This is a significant time commitment e.g. 2-4h usually.

**Comment by EilidhRoss1** [comment]
@rufuspollock Thanks for your comments. I'm going to propose that once wiser is more underway and we've got a better idea of our direction, offerings and budget we can perhaps budget in a certain amount of time for collaborating/connecting with other orgs. I will get in touch with Pierre-Alexandre and let him know that we currently don't have the bandwidth but that we hope to connect in the future. I will go  ahead with this unless you've got a different proposal of how we proceed with La Matrice?
